ecs-cli/modules/commands/configure/configure_command.go

```diff
@@ -25,7 +25,7 @@ import (
 func ConfigureCommand() cli.Command {
 	return cli.Command{
 		Name:   "configure",
-		Usage:  "Store a single cluster config.",
+		Usage:  "Configures a single cluster config.",
```

Comment:
Hmm, this wording sounds a bit off. I'd probably just stick with "Stores", since according to the [docs](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/cmd-ecs-cli-configure.html): "The resulting configuration is **stored** in the ~/.ecs/config file."